src/Constants.ts

```diff
@@ -51,7 +51,6 @@ export enum LANGUAGE {
 	APEX = 'apex',
 	JAVA = 'java',
```

Comment:
Can Java be removed?

Reply:
I guess we could. It would save us 1.14 MB. But do we really want to?